package.json

```diff
@@ -19,6 +19,7 @@
   "license": "ISC",
   "dependencies": {
     "camelcase": "1.0.2",
+    "co": "^4.6.0",
```

Comment:
usually good not to let packages autoupdate, we've had problems in the past with this